ballerina/CompilerPlugin.toml

```diff
@@ -3,4 +3,4 @@ id = "kafka-compiler-plugin"
 class = "io.ballerina.stdlib.kafka.plugin.KafkaCompilerPlugin"
 
 [[dependency]]
-path = "../compiler-plugin/build/libs/kafka-compiler-plugin-3.10.1.jar"
+path = "../compiler-plugin/build/libs/kafka-compiler-plugin-3.10.2-SNAPSHOT.jar"
```

Comment:
Why is it as snapshot here? Is it ok to have snapshot jars?

Reply:
yes. these will be automatically changed when we do a release